.yamato/config/universal_stereo.metafile

```diff
@@ -8,14 +8,6 @@ test_platforms:
   - type: editmode
   - type: Standalone
     name: Standalone_cache
-    utr_flags:
-    - [all]: compilation-errors-as-warnings
```

Comment:
is removal of `compilation-errors-as-warnings` intentional?

Reply:
Tbh I'm not sure about this one. I will add it back, thanks.

Reply:
Thinking again, I'm really not sure if we need this argument at run step... As the name implies, it should be used during the build/compilation step/job.
I found this:
```
--compilation-errors-as-warnings
    Fail a test suite if script compilation errors occurs. Temporary, until random compilation errors during editor startup are fixed
```